metaflow/plugins/aip/aip.py

```diff
@@ -1724,6 +1724,7 @@ def _get_aip_exit_handler_op(
                 " && python -m metaflow.plugins.aip.aip_exit_handler"
                 f" --flow_name {self.name}"
                 f" --run_id {METAFLOW_RUN_ID}"
+                f" --argo_workflow_uid {ARGO_WORKFLOW_UID}"
```

Comment:
Since you already have `METAFLOW_RUN_ID = f"argo-{ARGO_WORKFLOW_UID}"`, adding another ARGO_WORKFLOW_UID feels a bit redundant. Maybe we only keep one of them in the CLI?

Reply:
This CLI is not customer facing.. and this is easier with variables rather than adding logic to append `argo-` to ARGO_WORKFLOW_UID